### ISSUE TITLE
Support multiple profiles

### ISSUE THREAD

**Issue by lxalln** [allendevco/pill-logger#17]
Maybe give people options to represent themselves with emoji?
:dog: :balloon: :cat: :man: :woman: :baby: 

These could then be shown alongside each dosage row.

Would need to be able to filter by profile, showing a selection of at least one profiles.

**Comment by lxalln** [comment]
I forgot we had this issue here. I actually quite like the emoji idea still. I'm going to create a pull request where I can discuss progress.

**Comment by Dreagen** [comment]
That wouldn't work with the way I did it. You'd only ever see one emoji

**Comment by lxalln** [comment]
I agree, I just foresee a request from people to have a shared view of history. Which should be easy enough to do.

**Comment by Dreagen** [comment]
Yeah probably true
